src/components/UserUpdate/UserUpdate.tsx

```diff
@@ -182,41 +184,6 @@ const UserUpdate: React.FC<UserUpdateProps> = ({ id }): JSX.Element => {
                 </select>
               </label>
             </div>
-            <div>
-              <label>
-                {t('userType')}
-                <div className={styles.radio_buttons}>
-                  <input
-                    type="radio"
-                    id="admin"
-                    value="selectadmin"
-                    name="selectRole"
-                    checked={formState.selectedOption === 'selectadmin'}
-                    onChange={(e) => {
-                      setFormState({
-                        ...formState,
-                        selectedOption: e.target.value,
-                      });
-                    }}
-                  />
-                  <label htmlFor="admin">{t('admin')}</label>
-                  <input
-                    type="radio"
-                    id="superadmin"
-                    value="selectsuperadmin"
```

Comment:
Why have you removed the functionality to adjust the user type?

Reply:
@palisadoes  Because @anshgoyalevil pointed to remove that


> In that case, you can keep Default Language Field. Since User Type is still redundant. [(Comment Link)](url)

 also we have a separate page for managing user roles and change them.

Reply:
Also the userUpdateApi do not support this